.circleci/config.yml

```
@@ -30,21 +30,19 @@ jobs:
       - run: DOCKER_BUILDKIT=1 BUILDKIT_PROGRESS=plain BUILD_FROM_INTERMEDIATE=y make build build_serialize
       - restore_cache:
           keys:
-            - v1.3-inputdata-{{ checksum "download_inputdata.sh" }}-{{ checksum "requirements.txt"}}
+            - v1.4--{{ checksum "requirements.txt"}}
```

To match the below key.
```suggestion
            - v1.4-{{ checksum "requirements.txt"}}
```